### PR TITLE
Fixed text annotation comb input box

### PR DESCRIPTION
**Pull request by symtalha14**
I have tried to address issue #12865. Please look at it and suggest changes if any.

### CODE REVIEW

File: web/annotation_layer_builder.css [hunk 1]
```diff
@@ -140,7 +140,13 @@
    * area, causing horizontal scrolling. We avoid this by extending the width
    * when the element has focus and revert this when it loses focus.
    */
-  width: 115%;
+  /* width: 115%; */
+
+  /* 
+   * Modifying the width because it is making the textAnnotation input box
+   * overlap with other elements  
+   */
```
Comment (Snuffleupagus) [inline web/annotation_layer_builder.css:148]: You should never leave commented-out code in patches, and the added comment doesn't really add much value.

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]
Please don't include *unrelated* files in your patches, hence the `package-lock.json` must be removed.

Furthermore, have you successfully tested this manually in the viewer *and* do all of the tests pass with this patch?
Note that these changes are from PR #7649, and see also https://github.com/mozilla/pdf.js/wiki/Contributing#4-run-lint-and-testing

Finally, please provide a *meaningful* commit-message (and PR description) that actually describes the changes; note that "Solves issue #12865" isn't really helpful/useful in general.

File: web/annotation_layer_builder.css (hunk 1 above)
Comment (symtalha14) [reply web/annotation_layer_builder.css:148]: Thanks for the update.  Yes I have performed all the tests on it, working fine. I'll remove the comment then and the package-lock file.

**Comment by symtalha14** [comment]
So basically a CSS focus event style was modifying the input box width whenever the input box got focus. It modified the width from 100% to 115%. According to a comment in there, it was done to adjust the letter spacing of the characters in that input box. I have reduced the expanded width from 115% to 103% and it works fine now and passes all the test unit tests/reference image test.

File: web/annotation_layer_builder.css (hunk 1 above)
Comment (Snuffleupagus) [reply web/annotation_layer_builder.css]: Please note https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

Given that all of this code was written by Tim, I think it makes sense to ask him to review it (time permitting).

File: web/annotation_layer_builder.css [hunk 2]
```diff
@@ -140,7 +140,8 @@
    * area, causing horizontal scrolling. We avoid this by extending the width
    * when the element has focus and revert this when it loses focus.
    */
-  width: 115%;
+
```
Comment (Snuffleupagus) [inline web/annotation_layer_builder.css:143]: Unnecessary new-line, please revert.

---

Also, you've *still* included `package-lock.json` in this patch; in must be *completely* removed from the diff.

Comment (symtalha14) [reply web/annotation_layer_builder.css]: I have removed the package-lock.json file. Do let me know if you want me to close this pull request and create a new one to clarify things. Alright, waiting for @timvandermeij 's review.

Comment (Snuffleupagus) [reply web/annotation_layer_builder.css]: Sorry, but you're *still* including the `package-lock.json` file and you've not [squashed the commit](https://github.com/mozilla/pdf.js/wiki/Squashing-Commits) either.
It really shouldn't be necessary to create a new PR, please just edit the existing commit series to address these issues.

Comment (symtalha14) [reply web/annotation_layer_builder.css]: Hello, I have stashed the commits. And actually I have checked the "Load diff". It is showing me that package-lock.json was deleted.

Comment (Snuffleupagus) [reply web/annotation_layer_builder.css]: And, once more, please make sure that the *commit message* makes sense after you've squashed the commits (and edit it as required).

Comment (symtalha14) [reply web/annotation_layer_builder.css]: Done :)

**Comment by timvandermeij** [comment]
I did a quick check with my original PDF file and so far it looks like this works. Note that this is dynamic behavior that the tests can't cover, so because we have to rely on manual testing here I'd like to take some more time to play with more PDFs later to make sure it works in all scenarios.

**Comment by symtalha14** [comment]
I get your point

> I did a quick check with my original PDF file and so far it looks like this works. Note that this is dynamic behavior that the tests can't cover, so because we have to rely on manual testing here I'd like to take some more time to play with more PDFs later to make sure it works in all scenarios.

I get your point. I have checked in some sample PDFs from my side as well, seems fine. Let me know if you find any malfunction.

**Comment by symtalha14** [comment]
Hey, any updates?

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Thank you for your contribution!

**Pull request merged**